sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_param.json

```diff
@@ -44,6 +44,12 @@
     "applicationInsightsName": {
         "value": ""
     },
+    "logAnalyticsName": {
```

Comment:
same as above

Reply:
commented above, name is used for naming resource, arm id is used for (log analytics) workspaceId in app insights creation